ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -294,6 +294,17 @@ objects:
             --junit-dir /tmp/artifacts/junit -f -
         }
 
+        function run-csi-tests() {
+          for url in $(curl -s https://api.github.com/repos/openshift/ovirt-csi-driver-operator/contents/manifests | jq '.[].download_url'); do
```

Comment:
The operator starts up and spins the csi driver controller and nodes. We have a minor fail on the storage class apply - see below